boards/arm/lpcxpresso55s36/lpcxpresso55s36.dts

```diff
@@ -127,3 +128,10 @@
 	pinctrl-names = "default";
 	status = "ok";
 };
+
+&flexpwm1_pwm0 {
+	status = "okay";
+	pinctrl-0 = <&pinmux_flexpwm1_pwm0A
```

Comment:
pin control properties on LPC SOCs are group based, so an array of phandles for `pinctrl-0` properties should not be used here. You should create one `pinctrl-flexpwm1` group, and add the `PWM1_A0` and `PWM1_B0` pins to it